vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -42,6 +62,30 @@ void Create::Start(ledger::CreateWalletCallback callback) {
       ledger::URL_METHOD::GET, on_req);
 }
 
+void Create::StartSafetyNetCallback(
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers,
+    ledger::CreateWalletCallback callback) {
+
+  ledger_->LogResponse(__func__, response_status_code, response, headers);
+
+  unsigned int statusCode;
+  std::string error;
+  bool hasResponseError = braveledger_bat_helper::getJSONResponse(
```

Comment:
should be `snake_case`

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (19)